process/wg-lifecycle-documents/Vuln_Disc_wg_graduation_stage.md

```diff
@@ -0,0 +1,46 @@
+## Working Group graduation application
+Vulnerability Disclosures WG
+
+### WG has met all Incubating requirements
+  *  n/a
+
+### List of regular contributors
+The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
+- [Christopher "CRob" Robinson, Intel](https://github.com/SecurityCRob)
+- [Jonathan Leitschuh, Dan Kaminsky Fellowship - HUMAN](https://github.com/JLLeitschuh)
+- [Madison Oliver, GitHub Security Lab](https://github.com/taladrane)
+- [David A Wheeler, LF/OSSF](https://github.com/david-a-wheeler)
+- [Randall T. Vasquez (SKF/Gentoo/Homebrew)](https://github.com/ran-dall)
+- [Adolfo García Veytia, Chainguard & OpenVEX](https://github.com/puerco)
+- [Andrew Pollock, Google & OSV](https://github.com/andrewpollock)
+- [Arnaud Le Hors, IBM](https://github.com/lehors)
+- [Art Manion, ANALYGENCE](https://github.com/zmanion)
+- [Avishay Balter, Microsoft](https://github.com/balteravishay)
+- [Jay White, Microsoft](https://github.com/camaleon2016)
+- Jennifer Mitchell, Tidelift
+- [Ixchel Ruiz, JFrog](https://github.com/ixchelruiz)
+- [Marcus Meissner (SUSE)](https://github.com/msmeissn)
+- [Nathan Menhorn, AMD](https://github.com/nathan-menhorn)
+- [Nicole Schwartz, ActiveState](https://github.com/NicoleSchwartz/CircuitSwan)
+- Oliver Chang, Google & OSV
+- [Paulo Flabiano Smorigo (Ubuntu/Canonical)](https://github.com/pfsmorigo)
+- Yotam Perkal, Rezilion
+
+
+### Governance
```

Comment:
Can you please add a link in here that points to the project governance documents (per the #296 TIs Gives+Gets and updated WG template)?

Reply:
added link to wg charter - https://github.com/ossf/wg-vulnerability-disclosures/blob/main/CHARTER.md